.github/workflows/extension-release-published.yml

```diff
@@ -16,7 +16,7 @@ on:
       deployToMavenCentral:
         description: 'Specify it if you want to deploy to maven'
         required: false
-        default: 'true'
+        default: true
```

Comment:
this was failing @sayaliM0412  
![image](https://github.com/user-attachments/assets/c4b6943a-2d67-41f1-be59-dcdea7c627d1)